### ISSUE TITLE
As a user I want to be able to delete a log template and retain my logs of the same template so I only delete what I want

### ISSUE THREAD

**Issue by mcke0368** [mcke0368/CST8334#8]
* When a log template is deleted, all logs created from that template are deleted as well. This should not be
* Templates for logs can be deleted without affecting existing logs
* Peer review by at least 2 ppl (Not yourself) and merged in successfully.

**Comment by mcke0368** [comment]
/100

**Comment by BunnyJojo** [comment]
Database creation file adventureinnovation.sql  has foreign key constraint and index constraint on logbook_type.
remove 
ADD CONSTRAINT `logbook_ibfk_1` FOREIGN KEY (`logbook_type_id`) REFERENCES `logbook_type` (`id`) ON DELETE CASCADE,
 ADD KEY `logbook_type_id` (`logbook_type_id`),
Will keep the log books in the database

**Comment by amclean1980** [comment]
YEah, there are many mistakes in their database.  We should talk about this tomorrow.  I've update the index's to foreign keys and also added in a few missing foreign keys.  So, when the migrations get added you can just rebase on it.
